ui/src/Components/EditProfile.jsx

```diff
@@ -176,9 +195,9 @@ class CreateProfile extends React.Component {
     
     render() {
         const checkToolFile = ()=>{
-            if(this.state.toolPicUrl!==''){
+            if(this.state.profile.result.has_tools){
```

Comment:
This is a bug from last sprint - was able to fix this sprint. @kvptkr you do not need to worry about this for your task